packages/components/src/internal/components/labels/utils.ts

```diff
@@ -1,5 +1,5 @@
 import { User } from '../base/models/User';
 
 export function userCanPrintLabels(user: User): boolean {
-    return !user.isGuest;
+    return user && !user.isGuest;
```

Comment:
should add a quick jest test case to SamplesTabbedGridPanel for this with showLabelOption true but user as guest